app/frontend/src/components/apiAccess/ClientStatus.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div>
+    {{ env }}:
```

Comment:
Should this be in its own span object?

Reply:
No doesn't need to be inline or block level, just text